BCP00301Test.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2018 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+import json
+
+from GenericTest import GenericTest, NMOSTestException, NMOSInitException
+from Config import ENABLE_HTTPS
+from TestResult import Test
+
+BCP_API_KEY = "bcp-003-01"
+TMPFILE = "tls-report.json"
+
+
+class BCP00301Test(GenericTest):
+    """
+    Runs BCP-003-01-Test
+    """
+    def __init__(self, apis):
+        GenericTest.__init__(self, apis)
+        if not ENABLE_HTTPS:
+            raise NMOSInitException("BCP-003-01 can only be tested when ENABLE_HTTPS is set to True in Config.py")
+
+    def perform_test_ssl(self, test, args=None):
+        if os.path.exists(TMPFILE):
+            os.remove(TMPFILE)
+        if args is None:
+            args = []
+        try:
+            ret = subprocess.run(["testssl/testssl.sh", "--jsonfile", TMPFILE, "--warnings", "off"] + args +
+                                 ["{}:{}".format(self.apis[BCP_API_KEY]["hostname"], self.apis[BCP_API_KEY]["port"])])
+        except Exception as e:
+            raise NMOSTestException(test.FAIL("Unable to execute testssl.sh. Please see the README for installation "

```

Comment:
I think we could report this as `DISABLED`?

As for poor old Windows users, it does look like testssl.sh _can_ be made to work on Windows 10, though I haven't tried it yet. See https://github.com/drwetter/testssl.sh/issues/337, so perhaps no explicit OS check is necessary here.